swagger_spec_compatibility/rules/added_enum_value_in_response.py

```diff
@@ -35,4 +34,10 @@ def validate(cls, left_spec, right_spec):
                 continue
             if not is_path_in_top_level_paths(response_paths, enum_values_diff.path):
                 continue
-            yield cls.validation_message(format_path(enum_values_diff.path))
+            message = '\n \t\t{} {}: new enum value {} in property `{}`\n\t\t'.format(
```

Comment:
Please keeps PR as small and scoped as possible, it simplifies review and increases confidence around the change.
From what I read in the PR title we should be focusing on the creation of a new rule, and not on updating the wording of an already existing rule.

PS: removing this change would probably help you out with having green tests.